src/main/java/io/github/moehreag/axolotlclient/mixin/MinecraftClientMixin.java

```diff
@@ -96,9 +98,21 @@ public void stop(CallbackInfo ci){
     @Inject(method = "tick", at = @At("HEAD"))
     public void tickClient(CallbackInfo ci){
         Color.tickChroma();
-
         AxolotlClient.TickClient();
         DiscordRPC.update();
+        Zoom.tick();
+    }
+
+
+    @Redirect(method = "tick", at = @At(value = "INVOKE", target = "Lorg/lwjgl/input/Mouse;getEventDWheel()I"))
```

Comment:
I'd maybe put that in AxolotlClient.tickClient() (if using the tick method), but actually I'd rather go with a redirect mixin 
```suggestion
    @Redirect(method = "tick", at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/player/PlayerInventory;method_3134(I)V"))
```
to only zoom further in while no screen is opened

Reply:
I looked at the code, and it's only executed while `currentScreen == null` anyway.

The redirect also allows for cancellation of the scroll event.